app/script_variant.py

```diff
@@ -54,9 +55,10 @@ def romanize_japanese_location(word):
     if not word:
         return word
 
-    if word in JAPANESE_LOCATION_DICTIONARY:
-        yomigana = JAPANESE_LOCATION_DICTIONARY[word]
-        return jautils.hiragana_to_romaji(yomigana)
+    if (re.match(ur'([\u3400-\u9fff])', word)):
```

Comment:
This line appears twice. Can you extract this line into a function?

Also, can you add a comment what is [\u3400-\u9fff]?